tests/metrics/github/test_repos.py

```diff
@@ -1,20 +1,42 @@
 from datetime import date
 
+import pytest
+
+from metrics.github import repos
 from metrics.github.query import Repo
 
 
-def test_dont_filter_out_repos_from_unknown_orgs():
-    assert make_repo(org="other", name="any").is_tech_owned()
+@pytest.fixture
+def patch_query_to_return(monkeypatch):
+    def patch(the_repos):
+        monkeypatch.setattr(repos.query, "repos", lambda *_args: the_repos)
```

Comment:
> Why did you choose not to patch `query`?

I was copying the way previous code in these tests worked. I decided that I quite liked it because it depends only on a local implementation detail of `repos` (it calls a function `repos()` on an object `query`), not on which module that function is originally defined in. I don't feel strongly about that, though.

Reply:
> What are you trying to communicate by using `*_args` rather than `*args`?

This is a convention that I follow and which happily the IDE I use also follows: arguments whose name starts with underscores are unused. (My IDE (PyCharm) warns me about unused parameters unless they're named like that.)

I think that the convention originates in Haskell.